scripts/validate-requirements.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# bash scripts/validate-requirements.sh "END_USER_EMAIL" "ORGANIZATION_ID" "BILLING_ACCOUNT_ID"
+
+# -------------------------- Variables --------------------------
+# Expected versions of the installers
+TF_VERSION="1.2.1"
+GCLOUD_SDK_VERSION="319.0.0"
+GIT_VERSION="2.25.1"
+# User Inputs
+END_USER_CREDENTIAL="$1"
```

Comment:
you should use `getopts` to read the input from the user

https://github.com/terraform-google-modules/terraform-google-data-fusion/blob/52d8178c29cc2491d799e63ae1bea5a688b29773/modules/wait_healthy/wait_healthy.sh#L21

```
while getopts 'e:t:' c
do
  case $c in
    e) ENDPOINT=${OPTARG} ;;
    t) ACCESS_TOKEN=${OPTARG} ;;
    *)
      echo "Invalid flag ${OPTARG}"
      exit 1
      ;;
  esac
done
```